app/views/bookings/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<h2><%= @animal.name %></h2>
```

Comment:
Maybe add a h1 here to be clear that we're in the booking confirmed page (or it is the "my bookings" page?). This will help  the CSS work later.